federation-2/router-bridge/js-src/test_logger_worker.ts

```diff
@@ -2,14 +2,9 @@ declare namespace Deno {
   namespace core {
     function opAsync(opName: string, ...args: any[]): Promise<any>;
     const ops: Record<string, (...args: unknown[]) => any>;
-    function initializeAsyncOps(): void;
   }
 }
 
-// No async function can be called if this hasn't been done.
-// It needs to be done at runtime and cannot be snapshotted.
-Deno.core.initializeAsyncOps();
```

Comment:
the function does not exist anymore on the deno side. When upgrading deno, the tests were hanging because the js side was failing trying to execute that function, and the rust side never got an answer on the channel.
To know what to change, I looked at the diff between deno_core 0.171 and the latest:
https://github.com/denoland/deno/compare/v1.30.3...main#diff-f158991c36961c44552fb309384ee65ffe3d24d752d349b805572ee78b337f72L5